src/test/java/com/google/sps/data/PollingStationTest.java

```diff
@@ -38,6 +45,34 @@ public void tearDown() {
   // Test taking the JSON from the API response and creating a new PollingStation object
   @Test
   public void testFromJSONObject() throws Exception {
+    JSONObject pollingStationJsonObject =
+        new JSONObject(
+            "{\"id\": \"pollingId\","
+                + "\"address\": {\"locationName\": \"name\",\"line1\": \"1\","
+                + "\"line2\": \"2\",\"line3\": \"3\",\"city\": \"city\",\"state\": \"state\","
+                + "\"zip\": \"zip\"},\"pollingHours\": \"-\",\"name\": \"pollingStation\","
+                + "\"startDate\": \"start\",\"endDate\": \"end\", \"sources\": ["
+                + "{\"name\": \"Voting Information Project\", \"official\": true}]}");
+
+    PollingStation pollingStation =
+        PollingStation.fromJSONObject(pollingStationJsonObject, "pollingLocation");
+
+    Assert.assertEquals(pollingStation.getName(), "pollingStation");
+    Assert.assertEquals(pollingStation.getAddress(), "1 2 3, city, state zip");
+    Assert.assertEquals(pollingStation.getPollingHours(), "-");
+    Assert.assertEquals(pollingStation.getStartDate(), "start");
+    Assert.assertEquals(pollingStation.getEndDate(), "end");
+    Assert.assertEquals(pollingStation.getLocationType(), "pollingLocation");
+
+    List<String> sources = new ArrayList<String>();
+    sources.add("Voting Information Project");
```

Comment:
You can just do:

ImmutableList<String> sourcesList = ImmutableList.of("Voting Information Project"); 

Change it here and elsewhere.